### PR TITLE
Bug fix: editing and adding individual class not functional

### PR DESCRIPTION
**Pull request by Xarfo**
# Description

Bug fix for displaying an individual class with ability to delete and edit.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Change status

- [x] Complete, tested, ready to review and merge

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/Pages/Classes/SinglePage/SingleClassView.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function SingleClassView(props) {
 
   const completeClass = e => {
     e.preventDefault();
-    if (props.activeClasses.id === props.match.params.id) {
+    if (props.SingleClass.id === props.match.params.id) {
```
Comment (josefiaaa) [inline src/components/Pages/Classes/SinglePage/SingleClassView.js]: Is this supposed to be a capital S?

File: src/components/Pages/Classes/SinglePage/SingleClassView.js [hunk 2]
```diff
@@ -63,9 +63,9 @@ function SingleClassView(props) {
     subject,
     link,
     finished
-  } = props.activeClasses;
+  } = props.singleClass;
 
-  console.log("activeClasses", props.activeClasses);
+  console.log("singleClass", props.SingleClass);
```
Comment (josefiaaa) [inline src/components/Pages/Classes/SinglePage/SingleClassView.js]: Same here, I just don't see where it's imported.
Also above in lines 60-61, first name and last name columns have been changed to "name".

Comment (Xarfo) [reply src/components/Pages/Classes/SinglePage/SingleClassView.js]: Rectified. Should have been 'singleClass'

File: src/components/Pages/Classes/SinglePage/SingleClassView.js (hunk 1 above)
Comment (Xarfo) [reply src/components/Pages/Classes/SinglePage/SingleClassView.js]: Rectified. Thanks, Josi.

**Pull request merged**